server/js/app/wireless-debug.js

```diff
@@ -0,0 +1,241 @@
+/*
+ * @fileoverview This is the entrypoint for the application part of Wireless
+ * Debug.
+ */
+
+define([
+  'jquery',
+  'app/device-metric-graphs',
+  'app/util',
+  'datatables',
+], ($, MetricGrapher, Util) => class WirelessDebug {
+  /** Initialize the app */
+  constructor() {
+    /** @private @const {!jQuery} */
+    this.logTable_ = $('.log-table');
+
+    /** @private @const {!jQuery} */
+    this.metricsTable_ = $('.metrics-table');
+
+    /** @private @const {?WebSocket} */
+    this.ws_ = null;
+
+    /** @private @const {?MetricGrapher} */
+    this.metricGrapher = null;
+
+    /** @private @const {!DataTable} */
+    this.dataTable = $('#log-table').DataTable();
+  }
+
+  /**
+   * Opens a WebSocket connection to the server
+   */
+  start() {
+    this.ws_ = new WebSocket(`ws://${location.host}/ws`);
+    this.ws_.onopen = () => this.websocketOnOpen();
+    this.ws_.onmessage = (message) => this.websocketOnMessage(message);
+
+    let api_key = $('#api-key');
+    let device = $('#device');
+    let app = $('#app');
+    let time = $('#start-time');
+    let data_table = $('#historical-log-table').DataTable();
+
+    device.on('change', () => {
+      let chosen_device = device.val();
+      // Gets rid of old data but keeps table structure.
+      data_table.destroy();
```

Comment:
let data_table = $('#historical-log-table').DataTable();

is moved to line 42.